samples/04_gis_analysts_data_scientists/export_video_using_arcgis_api_for_python.ipynb

```diff
@@ -0,0 +1,318 @@
+{
```

Comment:
<p>assuming the gis used is</p><p>gis = GIS('https://pythonapi.playground.esri.com/portal',</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;'arcgis_python',</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;'amazing_arcgis_123',&nbsp;</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"your_enterprise_profile")</p><p> </p><p>the current code gives error:</p><pre class="ql-syntax" spellcheck="false">---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
&lt;ipython-input-3-9d32b45fbbad&gt; in &lt;module&gt;
     17 
     18 landsat_item = exact_search(gis, 'Multispectral Landsat', 'esri', 'Imagery Layer')
---&gt; 19 landsat = landsat_item.layers[0]
     20 landsat_item

AttributeError: 'NoneType' object has no attribute 'layers'

</pre><p> </p><p> </p><p>The searched item is on not from api_data_owner account, search by specifying the owner name also</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>The code is taken from the following notebook: <a href="https://developers.arcgis.com/python/sample-notebooks/detect-super-blooms-using-satellite-image-classification/" target="_blank">https://developers.arcgis.com/python/sample-notebooks/detect-super-blooms-using-satellite-image-classification/</a></p><p> </p><p>The issue is fixed in the current NB. Owner name (esri_livingatlas) is already there in the query:</p><p>landsat_item = exact_search(gis, 'Multispectral Landsat', 'esri_livingatlas', 'Imagery Layer')</p>

Reply:
<p>the link uses online account and this notebook indicates enterprise account, The item is not found </p>

Reply:
<p>fixed</p>